controller/controller.go

```diff
@@ -53,6 +58,13 @@ const (
 	loggerKeyVersion    = "version"
 )
 
+var (
+	defaultPauseAnnotations = map[string]string{
+		"cluster.x-k8s.io/paused":             "true",
+		"operatorkit.giantswarm.io/reconcile": "false",
```

Comment:
How about this?

```suggestion
		"operatorkit.giantswarm.io/paused": "true",
```

Reply:
Not a fan but I see that the consistency is striking.